docs/image/skymaps.rst

```diff
@@ -63,3 +63,72 @@ The sky map can be easily displayed with an image viewer, by calling ``skymap.sh
         counts = SkyMap.read(filename)
         counts.name = 'Counts'
         counts.show()
+
+
+Working with the cutout and paste method
+----------------------------------------
+
+The `~gammapy.image.SkyMap` class offers `.paste()` and `.cutout()`
```

Comment:
Maybe mark up  the `paste` and `cutout` method so that in the HTML version it's links directly to the methods?